spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DnnStorage.scala

```diff
@@ -104,17 +104,23 @@ private[tensor] class DnnStorage[T: ClassTag](size: Int) extends Storage[T] {
   @throws(classOf[IOException])
   private def readObject(in: ObjectInputStream): Unit = {
     in.defaultReadObject()
-    ptr = new Pointer(allocate(this.size))
-    val elements = in.readObject().asInstanceOf[Array[Float]]
-    Memory.CopyArray2Ptr(elements, 0, ptr.address, 0, size, DnnStorage.FLOAT_BYTES)
+    if (!_isReleased) {
+      ptr = new Pointer(allocate(this.size))
+      val elements = in.readObject().asInstanceOf[Array[Float]]
+      Memory.CopyArray2Ptr(elements, 0, ptr.address, 0, size, DnnStorage.FLOAT_BYTES)
+    } else {
+      ptr = new Pointer(allocate(this.size))
```

Comment:
if it's released, should we use a null for ptr?

Reply:
Yes. You are right. I'll check the `ModelBroadcast`. It should be the bug of last fix of quantized tensor memory leak.